src/middlewared/middlewared/plugins/webdav.py

```diff
@@ -153,19 +153,29 @@ class Config:
     async def upper(self, data):
         data['protocol'] = data['protocol'].upper()
         data['htauth'] = data['htauth'].upper()
+        if data['certssl']:
```

Comment:
I am not sure the correct way here is showing the full related object. Perhaps we should simply show the certificate id, being idempotent with the input data.

Reply:
I think we should display the whole object. This will mean remaining consistent with how we handle this in other services

Reply:
The other services should eventually be fixed to not do that.

Reply:
In that case, I'll have it updated asap ; )